src/metatrain/utils/additive/composition.py

```diff
@@ -43,12 +43,6 @@ def __init__(
         self.dataset_info = dataset_info
         self.atomic_types = sorted(dataset_info.atomic_types)
 
-        self.register_buffer(
```

Comment:
this might actually break the checkpoints =/

Reply:
I know and I've told @abmazitov

Reply:
We should get to a fast and reliable implementation as fast as possible

Reply:
Yes, sure